[BUG] Example import-2.0.x/config/examples/Mks/Robin doesn't compile on bugfix. Undefined pins

### Bug Description

Provided Configs for MKS Robin does not compile on current bugfix (using platfomio)
Error: 'FSMC_CS_PIN' was not declared in this scope

### My Configurations

[Configuration.zip](https://github.com/MarlinFirmware/Marlin/files/5276004/Configuration.zip)

### Steps to Reproduce

1. Download marlin bugfix
2. apply Config files and platformio.ini from attached Configuration.zip  (Same as import-2.0.x/config/examples/Mks/Robin)
3. Attempt to compile.

**Expected behavior:** 

It should compile

**Actual behavior:** 

Compile fails with this error:
```
Marlin/src/HAL/STM32F1/tft/tft_fsmc.cpp: In static member function 'static void TFT_FSMC::Init()':
Marlin/src/HAL/STM32F1/tft/tft_fsmc.cpp:89:16: error: 'FSMC_CS_PIN' was not declared in this scope
   uint8_t cs = FSMC_CS_PIN, rs = FSMC_RS_PIN;
                ^~~~~~~~~~~
Marlin/src/HAL/STM32F1/tft/tft_fsmc.cpp:89:16: note: suggested alternative: 'FSMC_CS_NE4'
   uint8_t cs = FSMC_CS_PIN, rs = FSMC_RS_PIN;
                ^~~~~~~~~~~
                FSMC_CS_NE4
Marlin/src/HAL/STM32F1/tft/tft_fsmc.cpp:118:11: error: 'rs' was not declared in this scope
   switch (rs) {
           ^~
Marlin/src/HAL/STM32F1/tft/tft_fsmc.cpp:118:11: note: suggested alternative: 'cs'
   switch (rs) {
           ^~
           cs
```
#### Additional Information

Issue is in pins_MKS_ROBIN.h block of code with the define is not active.

--- a/Marlin/src/pins/stm32f1/pins_MKS_ROBIN.h
+++ b/Marlin/src/pins/stm32f1/pins_MKS_ROBIN.h
@@ -143,6 +143,8 @@
   #define LCD_BACKLIGHT_PIN                 PG11
   #define FSMC_CS_PIN                       PG12  // NE4
   #define FSMC_RS_PIN                       PF0   // A0
+  #define TFT_CS_PIN                        FSMC_CS_PIN
+  #define TFT_RS_PIN                        FSMC_RS_PIN
 
   #define LCD_USE_DMA_FSMC                        // Use DMA transfers to send data to the TFT
   #define FSMC_DMA_DEV                      DMA2

--- a/Marlin/src/pins/stm32f1/pins_MKS_ROBIN.h
+++ b/Marlin/src/pins/stm32f1/pins_MKS_ROBIN.h
@@ -126,12 +126,7 @@
 #endif
 #define LED_PIN                             PB2
 
-#ifdef HAS_GRAPHICAL_TFT
-  #define TFT_RESET_PIN                     PF6
-  #define TFT_BACKLIGHT_PIN                 PG11
-  #define TFT_CS_PIN                        PG12  // NE4
-  #define TFT_RS_PIN                        PF0   // A0
-#else
+#ifdef HAS_FSMC_TFT
   /**
    * Note: MKS Robin TFT screens use various TFT controllers
    * Supported screens are based on the ILI9341, ST7789V and ILI9328 (320x240)
@@ -152,6 +147,11 @@
   #define LCD_USE_DMA_FSMC                        // Use DMA transfers to send data to the TFT
   #define FSMC_DMA_DEV                      DMA2
   #define FSMC_DMA_CHANNEL               DMA_CH5
+#elif HAS_GRAPHICAL_TFT
+  #define TFT_RESET_PIN                     PF6
+  #define TFT_BACKLIGHT_PIN                 PG11
+  #define TFT_CS_PIN                        PG12  // NE4
+  #define TFT_RS_PIN                        PF0   // A0
 #endif
 
 #if NEED_TOUCH_PINS

--- a/Marlin/src/pins/stm32f1/pins_MKS_ROBIN.h
+++ b/Marlin/src/pins/stm32f1/pins_MKS_ROBIN.h
@@ -126,7 +126,7 @@
 #endif
 #define LED_PIN                             PB2
 
-#ifdef HAS_FSMC_TFT
+#if HAS_FSMC_TFT
   /**
    * Note: MKS Robin TFT screens use various TFT controllers
    * Supported screens are based on the ILI9341, ST7789V and ILI9328 (320x240)

